Tests/SwiftProtobufTests/Test_AsyncMessageSequence.swift

```diff
@@ -136,18 +136,18 @@ final class Test_AsyncMessageSequence: XCTestCase {
   // Single varint describing a 2GB message
   func testTooLarge() async throws {
     let asyncBytes = asyncByteStream(bytes: [128, 128, 128, 128, 8])
-    var tooLargeThrown = false
     let decoded = asyncBytes.binaryProtobufDelimitedMessages(of: SwiftProtoTesting_TestAllTypes.self)
     do {
       for try await _ in decoded {
         XCTFail("Shouldn't have returned a value for an invalid stream.")
       }
     } catch {
-      if error as! BinaryDecodingError == .tooLarge {
-        tooLargeThrown = true
-      }
+      self.assertSwiftProtobufError(
+        error,
+        code: .binaryDecodingError,
+        message: "Message too large: Bytes and Strings have a max size of 2GB."
```

Comment:
Rather then duplicate the string, should it use `.BinaryDecoding.tooLarge`?

Reply:
Good catch, forgot to refactor this when I added the common error `static let`s.